CHANGELOG.md

```diff
@@ -74,6 +90,17 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
   - Updated granule write code to validate written createdAt is synced between
     datastores in cases where granule.createdAt is not provided for a new granule.
 
+## Fixed
+
+- **CUMULUS-3072**
```

Comment:
These logic changes were required for this PR to continue work, however these changes may be ported to main before this PR merges.